src/server.js

```diff
@@ -0,0 +1,150 @@
+"use strict";
+var http = require('http');
+var graphql = require('graphql');
+require('babel-register');
+var WebSocketServer = require('websocket').server;
+var graphql_tools = require('graphql-tools');
+var graphql_validator = require('graphql/validation');
+var graphql_execution = require('graphql/execution');
```

Comment:
I think you can just import validate and execute directly.
`import { execute } from 'graphql';`